projectsmember/Pages/CooperationRequest/Profiles/HamidForoughnia.html

```diff
@@ -0,0 +1,165 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title> رزومه</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-KK94CHFLLe+nY2dmCWGMq91rCGa5gtU4mk92HdvYe+M/SXH301p5ILy+dN9+nJOZ" crossorigin="anonymous">
```

Comment:
This is fine, but you might also find a similar file in this project from other directories, if so, please use those ones instead of downloading a new one. If not, this is just fine.